### PR TITLE
add classRating column in Class table

### PR DESCRIPTION
**Pull request by YoungsunCho**
#18 
getCalssinfo api pass isbooked variable to client
sorted class order by classRating

### CODE REVIEW

File: server/common/common/urls.py [hunk 1]
```diff
@@ -21,7 +21,7 @@
 
 urlpatterns = [
     path('admin/', admin.site.urls),
-    path('api/v1/',include('api.urls'))
+    path('api/v1/',include('api.urls')),
```
Comment (Ricci0708) [inline server/common/common/urls.py]: Do we need this comma in here?
I think that this element is the last elements of a list, so we do not need the comma.

File: server/common/api/models.py [hunk 2]
```diff
@@ -12,6 +12,7 @@ class Class(models.Model):
 	minGuestCount = models.IntegerField(default = 4)
 	maxGuestCount = models.IntegerField(default = 8)
 	price = models.FloatField()
+	classRating = models.FloatField(default = 0.0)
```
Comment (Ricci0708) [inline server/common/api/models.py]: In order to compute class rating, we need the number of class ratings.
If someone adds rating 3.0 on the class, how can we compute the effect of this rating?

Comment (Ricci0708) [inline server/common/api/models.py]: set the boundary of the class rating
https://docs.djangoproject.com/en/dev/ref/validators/#minvaluevalidator

File: server/common/common/urls.py (hunk 1 above)
Comment (YoungsunCho) [reply server/common/common/urls.py]: I edit it

File: server/common/api/models.py (hunk 2 above)
Comment (YoungsunCho) [reply server/common/api/models.py]: thanks to your advice!!
I set boundary from 0.0 to 5.0

Comment (YoungsunCho) [reply server/common/api/models.py]: add ratingCount column in class table

File: server/common/api/models.py [hunk 3]
```diff
@@ -12,6 +13,13 @@ class Class(models.Model):
 	minGuestCount = models.IntegerField(default = 4)
 	maxGuestCount = models.IntegerField(default = 8)
 	price = models.FloatField()
+	classRating = models.FloatField(default = 0.0,
+									validators = [
```
Comment (ethkim) [inline server/common/api/models.py]: Why are these lines indented so much? Should be the same level as "default = 0.0"

File: server/common/api/views.py [hunk 4]
```diff
@@ -88,17 +87,16 @@ def getClassList(request, date):
             for img in classImageList :
                 imageList.append(img.coverImage.url)
             jsondict["coverImage"] = imageList
```
Comment (ethkim) [inline server/common/api/views.py]: jsonDict?

Comment (YoungsunCho) [reply server/common/api/views.py]: changed naming to classList

**Pull request merged**